src/main/webapp/js/issues-detail.js

```diff
@@ -1,45 +1,39 @@
-/* global jQuery, view */
+/* global jQuery, view, renderZoomableTrendChart */
 (function ($) {
-    if ($('#severities-chart').length) {
-        initializePieCharts();
-    }
-
     redrawTrendCharts();
     storeAndRestoreCarousel('trend-carousel');
+    storeAndRestoreCarousel('overview-carousel');
 
     /**
      * Create a data table instance for all tables that are marked with class "property-table".
      */
-    $('table.property-table').DataTable({
-        pagingType: 'numbers', // Page number button only
-        columnDefs: [{
-            targets: 'no-sort', // Columns with class 'no-sort' are not orderable
-            orderable: false
-        }]
+    const propertyTables = $('table.property-table');
+    propertyTables.each(function () {
+        const table = jQuery3(this);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['jQuery3' is not defined.](https://app.codacy.com/manual/uhafner/warnings-ng-plugin/pullRequest?prid=4663559)